jsonmerge_utils.py

```diff
@@ -53,10 +53,9 @@ def configure_logger(log_level):
     """
     configures the logger object
     """
-    logging.basicConfig(filename='output.log')
+    logging.basicConfig(filename='output.log',level = log_level)
     if not logger.handlers:
         # Prevent logging from propagating to the root logger
-        logger.setLevel(log_level)
         logger.propagate = 0
         log_formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
         stream_handler = logging.StreamHandler()

```

Comment:
we can remove the logging.basicConfig line for now, since we are not focusing on logging to a file for now
add this after `stream_handler = logging.StreamHandler()`  line
```
logger.setLevel(logging.DEBUG)
stream_handler.setLevel(log_level)

```
Let me know if this works

Reply:
Yup the program runs fine with that, just pushed the new changes.